src/lib/LUA/lua.h

```diff
@@ -37,9 +40,19 @@ struct luaItem_selection {
     const char* const units;
 } PACKED;
 
+enum luaCmdStep_e : uint8_t {
+    lcsNone = 0,
+    lcsClick = 1,       // user has clicked the command to execute
+    lcsExecuting = 2,   // command is executing
+    lcsAskConfirm = 3,  // command pending user OK
+    lcsConfirmed = 4,   // user has confirmed
+    lcsCancel = 5,      // user has requested cancel
+    lcsQuery = 6,       // UI is requesting status update, only sent by Crossfire Config
```

Comment:
What does `only sent by Crossfire Config` mean? Because Query/Poll is sent from our LUA as well [on line 618](https://github.com/ExpressLRS/ExpressLRS/blob/094d15b52fcd78d6f54bab697aa4cb9663e5d5e0/src/lua/elrsV2.lua#L618)

Reply:
Oh there it is! I looked all over our lua and didn't see it, I thought our Lua only sent 5 over and over again since the field.state is never updated to 6. I knew for a fact that Crossfire Config did though. I'm also not sure about this [6 here](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/lib/LUA/lua.cpp#L300). I think that's lcsQuery but I'm not fully sure when that code will actually fire. When we're sending a command status in response to a query that is longer than the chunk length?

Reply:
From memory, that second one is used when sending long response packets. The crossfire LUA protocol expects a poll/query for each seperate chunk of the response.

Reply:
Yeah I know about the chunks but normally a chunk is handled by sending a READ request with the ID and chunk number, not a WRITE request setting to 6. I don't think we do this anywhere do we?

Reply:
0x2D is the write request, it writes a query. Thats the line 618 I pointed to earlier 😄

Reply:
Ah of course! The lack of ChunkId threw me off. You're really making me feel like a fool today, I appreciate you setting me straight. I've updated the mystery 6 with lcsQuery